source/game/p_client.cpp

```diff
@@ -798,6 +800,30 @@ void ClientBegin( edict_t *ent )
 	G_Gametype_ScoreEvent( client, "enterGame", NULL );
 }
 
+void ClientAuth ( edict_t *ent, uint64_t steamid )
+{
+	gclient_t *client = ent->r.client;
+	client->steamid = steamid;
+	client->authenticated = true;
+
+	if ( g_permanent_operators->string[0] ){
```

Comment:
```suggestion
	if ( g_permanent_operators->string[0] && steamid ){
```

Otherwise a misconfigured `g_permanent_operators` would allow people without a steamid to become operators

Reply:
you would be kicked before that if you didn't have a steamid